src/io/syscall.cr

```diff
@@ -81,6 +81,34 @@ module IO::Syscall
     end
   end
 
+  def writev_syscall_helper(slices, errno_msg : String) : Nil
```

Comment:
`errno_msg` argument here is IMHO pointless, why not just use `Error writing to IO`?